packages/lib/har/harUtils.ts

```diff
@@ -87,19 +95,45 @@ export const fromHarContent = (content?: HarFormatContent) => {
   if (content?.text) {
     return Buffer.from(content.text, content.encoding === 'base64' ? 'base64' : 'binary');
   }
+  if (content?.json) {
+    return content?.json;
```

Comment:
```suggestion
    return Buffer.from(stringifyPretty(content.json), 'utf8');
```
`fromHarContent` is supposed to always return a `Buffer` or a `string` because it is [assigned to](https://github.com/AmadeusITGroup/kassette/blob/master/packages/app/mocking/impl.ts#L788) `Payload.body` (which is defined in [packages/app/mocking/model.ts](https://github.com/AmadeusITGroup/kassette/blob/master/packages/app/mocking/model.ts#L583)).

Note that, because of this, the current code of the PR has an issue in case `content.json` is a simple string (which is a valid JSON, as allowed by [rfc8259](https://datatracker.ietf.org/doc/html/rfc8259#section-2)). The body of the response would then be the content of the string, instead of being properly enclosed in quotes, as required by the JSON format.